Wikipedia/Code/EditFunnel.swift

```diff
@@ -96,7 +96,7 @@
             return eventData
         }
 
-        let requiredData: [String: Any] = ["session_token": sessionID, "anon": isAnon, "app_install_id": appInstallID, "client_dt": timestamp]
```

Comment:
@mdholloway let's keep this `client_dt` for now - `EditFunnel` still flows only through the old legacy system, only `EditHistoryCompareFunnel` will have the call sites to the new system.

Reply:
Whoops! Thanks for catching that. Updated.